bin/build

```diff
@@ -36,13 +44,25 @@ fi
 set -e
 
 pushd $IMAGE_DIR
+  
+  if [[ -n "$GIT_TOKEN" ]] ; then
+    SECRETS_CONTAINER=$(docker run -d --rm busybox /bin/sh -c \
+      "echo $GIT_TOKEN > git_token.txt && httpd -p 8000 -h / && sleep 600")
+  fi
+
   cmd="docker build --tag $REPO/manageiq-base:$TAG \
                     --build-arg MIQ_REF=$MIQ_REF \
                     --build-arg SUI_REF=$SUI_REF\
                     --build-arg APPLIANCE_REF=$APPLIANCE_REF \
                     --build-arg MIQ_ORG=$MIQ_ORG \
                     --build-arg SUI_ORG=$SUI_ORG \
-                    --build-arg APPLIANCE_ORG=$APPLIANCE_ORG"
+                    --build-arg APPLIANCE_ORG=$APPLIANCE_ORG \
+                    --build-arg GIT_HOST=$GIT_HOST \
+                    --build-arg REPO_PREFIX=$REPO_PREFIX"
+
+  if [[ -n "$GIT_TOKEN" ]] ; then
+    cmd+=" --build-arg GIT_TOKEN_AVAILABLE=true --network=container:secrets-server"
```

Comment:
If we avoid the container above then we will also be able to just key off of the presence of $GIT_TOKEN rather than creating a separate build arg to indicate that it is available.